### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: odyssey/templates/deployment-odyssey.yaml [hunk 1]
```diff
@@ -21,11 +24,6 @@ spec:
         {{- with .Values.additionalAnnotations }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
-      labels:
-        {{- include "odyssey.selectorLabels" . | nindent 8 }}
```
Comment (p-tombez) [inline odyssey/templates/deployment-odyssey.yaml:25]: This part has not been moved like the other duplicate parts. Is it wanted ?

File: odyssey/values.yaml [hunk 2]
```diff
@@ -7,20 +7,38 @@ imagePullSecrets:
...
+volumes: []
+# extra volumes mounts for deployment
+volumeMounts: []
 
 image:
   repository: ghcr.io/camptocamp/odyssey
   tag: latest
   pullPolicy: Always
 
+additionalAnnotations: {}
```
Comment (p-tombez) [inline odyssey/values.yaml:35]: missing default empty `additionalLabels` value

**Review by p-tombez — CHANGES_REQUESTED** [review]

File: odyssey/templates/deployment-odyssey.yaml (hunk 1 above)
Comment (Vampouille) [reply odyssey/templates/deployment-odyssey.yaml:25]: yes "selectorLabels" are included in "labels"

File: odyssey/values.yaml (hunk 2 above)
Comment (Vampouille) [reply odyssey/values.yaml:35]: done with #107640a

**Review by p-tombez — APPROVED** [review]

**Pull request merged**